src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -61,14 +61,9 @@ namespace winrt::TerminalApp::implementation
     // - sender, e: not used
     void TerminalTab::_BellIndicatorTimerTick(Windows::Foundation::IInspectable const& /*sender*/, Windows::Foundation::IInspectable const& /*e*/)
     {
-        auto weakThis{ get_weak() };
-
-        if (auto tab{ weakThis.get() })
-        {
-            tab->ShowBellIndicator(false);
-            tab->_bellIndicatorTimer.value().Stop();
-            tab->_bellIndicatorTimer = std::nullopt;
-        }
+        ShowBellIndicator(false);
+        _bellIndicatorTimer->Stop();

```

Comment:
before we interact with the bell timer optional, we may want to make sure it `has_value`. it's safer that way in case one DOES end up getting scheduled multiple times

Reply:
Makes sense, done!